.circleci/config.yml

```diff
@@ -3,11 +3,8 @@ version: 2.1
 orbs:
   browser-tools: circleci/browser-tools@1.1
 
-  # Always take the latest version of the orb, this allows us to
-  # run specs against Solidus supported versions only without the need
-  # to change this configuration every time a Solidus version is released
-  # or goes EOL.
-  solidusio_extensions: solidusio/extensions@volatile
+  # Stick to the current major, so we won't get disrupting updates
+  solidusio_extensions: solidusio/extensions@0
```

Comment:
Do we still need this? Is the orb broken?

Reply:
Well, I have mixed feelings, "volatile" is really discouraged, but we're also in pre-1.0 mode… I'll revert for now, and we'll think of releasing 1.0 on the ORB and handle changes more "semantically" after that.

Reply:
Agreed